sky/cli.py

```diff
@@ -1582,6 +1582,8 @@ def cancel(cluster: str, all: bool, jobs: List[int]):  # pylint: disable=redefin
         core.cancel(cluster, all, jobs)
     except ValueError as e:
         raise click.UsageError(str(e))
+    except (exceptions.NotSupportedError, exceptions.ClusterNotUpError) as e:
+        click.echo(str(e))
```

Comment:
Do we want to use the following instead?
```
with ux_utils.print_exception_no_traceback():
    raise e
```

Reply:
Got it! That makes sense. Then I think it might be better to use `raise click.UsageError` as the L1584 did or directly `sys.exit(1)`? Reason: the return code may need to represent the status of the command.

Reply:
Good call! Opted for sys.exit(1). `click.UsageError` prints some context that suggests the usage of the CLI went wrong:
```
» sky cancel sky-abbf-zonghengasd 1
Usage: sky cancel [OPTIONS] CLUSTER [JOBS]...
Try 'sky cancel -h' for help.

Error: Cluster 'sky-abbf-zonghengasd' does not exist; skipped.
```